client/landing/stepper/declarative-flow/onboarding.ts

```diff
@@ -114,7 +115,12 @@ const onboarding: Flow = {
 
 		if ( isGoalsAtFrontExperiment ) {
 			// Note: these steps are not wrapped in `stepsWithRequiredLogin`
-			steps.unshift( STEPS.GOALS, STEPS.DESIGN_SETUP, STEPS.DIFM_STARTING_POINT );
+			steps.unshift(
+				STEPS.GOALS,
+				STEPS.DESIGN_CHOICES,
```

Comment:
This loads the new step. It's safe to add the step here even if the user never sees it (e.g. the user isn't in the experiment cohort).